src/base/application.h

```diff
@@ -432,6 +437,9 @@ class Application
     //! Current vector of input directories to search for input files
     std::vector<std::string> inputDirs;
 
+    //! Versions of Python to consider when attempting to load user extensions
+    vector<string> m_pythonSearchVersions = {"3.11", "3.10", "3.9", "3.8"};
```

Comment:
what about making this `set<string>` instead? When order doesn't matter, sets are much more convenient.

Reply:
The order here isn't irrelevant, though. I want to start with the most recent supported Python version and only look for older versions if the first one doesn't work.